js/newsletter.js

```diff
@@ -71,17 +71,19 @@ class Newsletter {
 			buttonDataTrackable = 'newsletter-unsubscribe';
 			buttonText = this.newsletterButton.innerHTML.replace(
 				'One-Click Sign Up<span class="n-util-visually-hidden">&nbsp;to',
-				'Unsubscribe<span class="n-util-visually-hidden">&nbsp;from'
+				'Subscribed<span class="n-util-visually-hidden">&nbsp;from'
 			);
```

Comment:
A problem here about accessibility. before this change the readup text for the button was 'Unsubscribe from newsletter title'